extra-requirements.txt

```diff
@@ -72,8 +72,8 @@ jsonschema:                 cicd
 dgl:                        cicd
 black==20.8b1:              test
 matplotlib:                 devel
-kubernetes>=18.20.0:        devel
-portforward>=0.2.4:         devel
```

Comment:
i am not sure about this. I would still keep in devel since this is for debugging and not needed in any image

Reply:
thanks done